lib/models/Compare_Crypto.rb

```diff
@@ -27,13 +26,20 @@ def display_mini_menu
         fiat_input = get_fiat_input
         puts CLEAR
         MainMenu.new.show_logo
-        set_crypto_1(crypto_inputs[0],fiat_input)
-        set_crypto_2(crypto_inputs[1],fiat_input)
+        crypto_inputs.each do |n|
```

Comment:
When using an iterator like `.each` , its good practice to be descriptive with the naming of the individual element. Rather than using  `|n|` , you could use `|crypto_element|` or something more specific to the purpose of the method